src/app/view/components/main-legacy/main-legacy.component.html

```diff
@@ -13,6 +13,7 @@
             <dot-main-nav></dot-main-nav>
         </aside>
         <main class="layout__main">
+            <dot-message-display></dot-message-display>
```

Comment:
This is a floating thing... you should move next to the `<dot-message-display></dot-message-display>`